boards/msba2/include/board.h

```diff
@@ -55,6 +55,15 @@ extern "C" {
 #define XTIMER_SHOOT_EARLY  3
 /** @} */
 
+/**
+ * @brief   Definition of the interface to the CC1100 radio
+ */
+#define CC110X_PARAMS_BOARD     {   .spi  = SPI_DEV(0),      \
+                                    .cs   = GPIO_PIN(1, 21), \
+                                    .gdo0 = GPIO_PIN(0, 27), \
+                                    .gdo1 = GPIO_PIN(1, 23), \
+                                    .gdo2 = GPIO_PIN(0, 28) }
```

Comment:
I've taken these ports from the [schematics linked on the MSBA2 wiki page](https://github.com/RIOT-OS/RIOT/wiki/Board%3A-MSBA2#more-info). This checks out with the values [found in the original definition](https://github.com/RIOT-OS/RIOT/pull/8437/files#diff-426dc6b9b80872c24335351f303b5eefL32) when you run them through the [expression GPIO_PIN() evaluates to](https://github.com/RIOT-OS/RIOT/blob/master/cpu/lpc2387/include/periph_cpu.h#L57).